views/partials/navbar.hbs

```diff
@@ -82,35 +112,64 @@
     <ul class='mobile-navbar-list' id='mobile-navbar-list'>
         {{#if onAdmin}}
 
-            <!-- APPLICATION REVIEW -->
-            <li><a href='/admin/application-review'>Application Review</a></li>
+            {{!-- Uncomment ahead of hackathon start --}}
+            {{!-- <!-- TEAMS -->
+            <li><a href='/teams'>Teams</a></li>
+
+            <!-- TEAM CREATION -->
+            <li><a href='/teams/create'>Team Creation</a></li>
+
+            <!-- PARTS BY TEAM -->
+            <li><a href='/parts'>Parts by Team</a></li> --}}
+
+            {{!-- Comment after applications close --}}
+            <!-- PARTS CREATION -->
+            <li><a href='/create'>Parts Creation</a></li>
+
+            <!-- APPLICATIONS -->
+            <li><a href='/admin/application-review'>Applications</a></li>
 
             <!-- MAILING LISTS -->
             <li><a href='/admin'>Mailing Lists</a></li>
 
             <!-- DASHBOARD -->
             <li><a href='/dashboard'>Me</a></li>
 
+            <!-- LOG OUT -->
+            <li class='center-flex'><a href='/logout'><button>Log Out</button></a></li>
+
         {{/if}} {{#if onDashboard}}
 
+            {{!-- Comment after applications close --}}
             <!-- APPLICATIONS -->
-            {{!-- <li><a href='/dashboard/application'>Application</a></li> --}}
+            <li><a href='/dashboard/application'>Application</a></li>
+
+            {{!-- Uncomment ahead of hackathon start --}}
+            {{!-- <!-- PARTS -->
+            <li><a href="/dashboard/parts">Parts</a></li>
+
+            <!-- MY PARTS -->
+            <li><a href="/dashboard/me/parts">My Parts</a></li>
 
-            <!-- BOXES -->
-            <li><a href='/boxes'>Boxes</a></li>
+            <!-- MY TEAM -->
+            <li><a href="/dashboard/me/parts">My Team</a></li>
 
             <!-- RESOURCES -->
-            <li><a href="/dashboard/resources">Resources</a></li>
+            <li><a href="/dashboard/resources">Resources</a></li> --}}
 
             <!-- DASHBOARD -->
             <li><a href='/dashboard/me'>Me</a></li>
 
+            <!-- LOG OUT -->
+            <li class='center-flex'><a href='/logout'><button>Log Out</button></a></li>
```

Comment:
You could remove `center-flex` from the log in button, it would improve visual consistency since all the other links are left-aligned, but only the button is center-aligned.

Reply:
Yeah, that's fair. Not quite sure why I felt so strongly about positioning it that way.